setup.py

```diff
@@ -11,7 +11,7 @@
     "scipy~=1.7.3",
     "statsmodels~=0.13.2",
     "tabulate~=0.8.10",
-    "z3_solver~=4.8.13.0",
+    "z3_solver>=4.11.2",
```

Comment:
Is there a reason to swap from the `Compatible release clause` `~=` to the greater or equal to clause `>=` ?

Reply:
I didn't 100% understand what `~=` meant. After a quick google, I'll swap it to that.